chat-server/src/services/conversations.ts

```diff
@@ -132,10 +160,7 @@ export interface ConversationsService {
   }: RateMessageParams) => Promise<boolean>;
 }
 
-/**
- OSS_TODO: make these configurable from the entry point. though i think these messages are reasonable defaults
- */
-export const conversationConstants = {
+export const defaultConversationConstants = {
```

Comment:
```suggestion
export const defaultConversationConstants: ConversationConstants = {
```

nitty but it'll make sure our default is aligned with the type def